scripts/linters/js_ts_linter.py

```diff
@@ -284,6 +284,54 @@ def _check_any_type(self):
 
         return [summary_message]
 
+    def _check_http_requests(self):
+        """Checks if the http requests are made only by
+        backend-api.service.ts.
+        """
+
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting HTTP requests check')
+            python_utils.PRINT('----------------------------------------')
+
+        http_client_pattern = r':\n? *HttpClient'
+
+        excluded_files = [
+            'core/templates/services/request-interceptor.service.spec.ts'
+        ]
+
+        with linter_utils.redirect_stdout(sys.stdout):
+            failed = False
+
+        for file_path in self.all_filepaths:
```

Comment:
Are we expecting this block to be inside the `with` block?

Reply:
done